src/rpc/server/scala/src/main/scala/com/ctc/omega_edit/grpc/Session.scala

```diff
@@ -194,15 +194,15 @@ class Session(
       }
 
     case Insert(data, offset) =>
-      session.insert(data, offset) match {
+      session.insert(data.toByteArray(), offset) match {
```

Comment:
Can the data just come in as a ByteArray?

Reply:
I think by default it comes in a protobuf ByteString

Reply:
oops!

Reply:
> oops!

I assume by this reply I was thinking of the wrong scenario lol